### PR TITLE
Add instruction to add environment variables in Windows

### PR DESCRIPTION
**Pull request by andrew-hwahin**

### CODE REVIEW

**Comment by jtoar** [comment]
Hey @andrew-hwahin thanks for improving the contributor DX for Windows users here! Anything to make that process easier goes a long way. Anything we can do to help you get this one merged?

**Comment by andrew-hwahin** [comment]
Hi, @jtoar & @Tobbe I've updated the casing of the command prompt. I think all Windows users shall be able to set the environment variable by using that method because most Windows comes with the Command Prompt.

**Comment by Tobbe** [comment]
I was expecting something like this:

_On Windows_

For Cmd:

```
setx RW_PATH "C:\Users\mojombo\path\to\redwood\framework"
```

For PowerShell:

```
[System.Environment]::SetEnvironmentVariable("RW_PATH", "C:\Users\mojombo\path\to\redwood\framework",[System.EnvironmentVariableTarget]::User)
```

For gitbash:

Use Cmd or PowerShell to set a regular Windows environment variable. gitbash will read those and merge with its own variables. Or you can add the following to your `.bashrc` file

```
export RW_PATH=/c/Users/mojombo/path/to/redwood/framework
```

Or we could just tell everyone, no matter what shell, to use the built-in GUI for setting env vars

image

Or, final option, which I feel stupid for not realizing until I started writing this reply, we can mention that the Cmd option works no matter what shell you normally use, you just have to switch to Cmd for running this command, then you can switch back to your shell of choice.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -245,12 +245,30 @@ yarn rwt copy:watch
...
+For PowerShell:
+
+```terminal
+[System.Environment]::SetEnvironmentVariable("RW_PATH", "C:\Users\mojombo\path\to\redwood\framework",[System.EnvironmentVariableTarget]::User)
+```
+
+For gitbash:
+
+```terminal
+export RW_PATH=/c/Users/mojombo/path/to/redwood/framework
```
Comment (Tobbe) [inline CONTRIBUTING.md:265]: Did you try this? I'm not sure it's persisted when you do it like that.

**Comment by thedavidprice** [comment]
Hi @andrew-hwahin and @Tobbe Just checking in about the status of this one — is this something we'd still like to do?

**Comment by Tobbe** [comment]
> is this something we'd still like to do?

Yes, I think we should. @andrew-hwahin Want to give it another go, taking my latest comments into consideration?

**Comment by andrew-hwahin** [comment]
@Tobbe @thedavidprice sorry for the late response, I'm not quite sure about it, I did some research but unable to come out with a concrete answer, I think I shall pass this on... sorry

**Pull request closed without merging**